pkg/types/aws/defaults/platform.go

```diff
@@ -72,3 +78,14 @@ func InstanceTypes(region string, arch types.Architecture, topology configv1.Top
 		}
 	}
 }
+
+// SkippedZones returns the list of supported zones.
+func SkippedZones(region string, zones []string) []string {
```

Comment:
```suggestion
func SupportedZones(region string, zones []string) []string {
```
I think this is a better name